qmi/instruments/picotech/_picoscope.py

```diff
@@ -352,3 +354,7 @@ def get_block_data(self, channels: List[int]) -> Tuple[np.ndarray, float, List[b
     @rpc_method
     def run_block(self, num_pretrig_samples: int, num_posttrig_samples: int, timebase: int) -> None:
```

Comment:
would change `timebase` to `time_base` here as well

Reply:
ok